NebulaWorld/SimulatedWorld.cs

```diff
@@ -607,6 +619,26 @@ public static void RenderPlayerNameTagsInGame()
 
                     // Make sure the text is pointing at the camera
                     playerNameText.transform.rotation = GameCamera.main.transform.rotation;
+
+                    // Resizes the text based on distance from camera for better visual quality
+                    var distanceFromCamera = Vector3.Distance(playerNameText.transform.position, GameCamera.main.transform.position);
+                    var nameTextMesh = playerNameText.GetComponent<TextMesh>();
+
+                    if (distanceFromCamera > 100f)
```

Comment:
IMO some kind of linear interpolation would look better than the jumpy steps that happen here

Reply:
Something like this?

```cs
nameTextMesh.characterSize = Mathf.Lerp(0.1f, 0.2f, Mathf.Clamp(distanceFromCamera / 100, 0, 1));
nameTextMesh.fontSize = (int)Mathf.Lerp(36, 60, Mathf.Clamp(distanceFromCamera / 100, 0, 1));
```